oasislmf/utils/inputs.py

```diff
@@ -22,25 +23,43 @@ class InputValues(object):
     """
     def __init__(self, args, update_keys=True):
         self.logger = logging.getLogger()
-
         self.args = args
         self.config = {}
         self.config_fp = self.get('config', is_path=True)
         self.config_mapping = get_config_profile()
 
         if self.config_fp is not None:
-            self.config = self.load_config_file()
-            self.config_dir = os.path.dirname(self.config_fp)
+            try:
+                self.config = self.load_config_file()
+                self.config_dir = os.path.dirname(self.config_fp)
+                self.list_unknown_keys()
+            except JSONDecodeError as e:
+                raise OasisException('Error: Invalid configuration file "{}"'.format(self.config_fp), e)
```

Comment:
Suggestion for the error message:
f"Configuration file {self.config_fp} is not a valid json file. {e.msg}"